bee-message/src/output/unlock_condition/timelock.rs

```diff
@@ -40,3 +48,39 @@ impl TimelockUnlockCondition {
         self.timestamp
     }
 }
+
```

Comment:
I think you can derive this and use a function that receives `impl Deref<Target=u32>` in `verify_with`.

Reply:
The verification needs 2 fields

Reply:
but the verification checks that each field is not zero. If you verified each field with a different function it would produce the same code.

Reply:
It checks that they are not BOTH zero at the same time, one of them is allowed to be 0

Reply:
ohhhhhhhhhhh damn, nevermind then